reformat_gherkin/formatter.py

```diff
@@ -268,6 +263,25 @@ def __find_nodes_with_newline(self) -> Set[Node]:
 
         return nodes_with_newline
 
+    def __find_max_step_keyword_width(self) -> int:
+        """
+        Find the length of the longest step keyword in the document. This is
+        used for aligning step keywords.
+        """
+        if self.step_keyword_alignment == AlignmentMode.NONE:
```

Comment:
Comparison with enum should be by identity: `is` over `==`.

Reply:
You learn something new every day. :) Will change that.